source/config/__init__.py

```diff
@@ -738,7 +742,7 @@ def renameProfile(self, oldName, newName):
 		if oldName.lower() != newName.lower() and os.path.isfile(newFn):
 			raise ValueError("A profile with the same name already exists: %s" % newName)
 
-		os.rename(oldFn, newFn)
+		osreplace.replace(oldFn, newFn)

```

Comment:
@jcsteh As I read it, it could be possible this method is called with the same oldName and newName, probably we should check that and just return early instead. Then I agree it's not nescesary as far as I can see.